README.md

```diff
@@ -201,6 +201,219 @@ hints and first-parties can use `Feature-Policy` to decide which third-parties t
 privilege with detailed user agent information. In any event, moving to this opt-in model means
 that the extent of a site's usage can be monitored and evaluated.
 
+# Use-cases
+
+This section attempts to document the current uses for the `User-Agent` string,
+and how similar functionality could be enabled using UA-CH.
+
+## Differential serving based on browser features
+This use-case enables services like [polyfill.io](https://polyfill.io) to serve
+custom-tailored polyfills to their users, without bloating up the experience of
+modern browser users.
+Similarly, when serving Javascript to users, one can avoid
+transpilation (which can result in bloat and inefficient code) for browsers
+that support the latest ES features that were used.
+Finally, when serving images, some browsers don't update their `Accept` request
+headers, while in other cases (*cough* WebP *cough*) the MIME type is not
+descriptive enough to distinguish between different variants of the same
+format. In those cases, knowing the browser and its version can be critical to
+serving the right image variant. 
+
+For that use case to work, the server needs to be aware of the browser and its
+meaningful version, and map that to a list of available features. That enables
+it to know which polyfill or code variant to serve.
+
+Services that wish to do that using UA-CH will need to inspect the `Sec-CH-UA`
+header, that is sent by default on every request, and modify their reponse
```

Comment:
This assumes that `Sec-CH-UA` accurately reports the browser without any extra opt-in, which some other issues have up in the air. I think this section should list the possible designs, and say how each does or doesn't make the use case possible.

> * If `Sec-CH-UA` can randomly include a different browser's identifier, this use case becomes impossible to support.
> * If `Sec-CH-UA` only includes the browser engine by default, services like `polyfill.io` need to opt into the full UA with `Accept-CH: UA` and inspect the result to determine what response to send.
>    * If the browser supports a TLS extension to opt into client hints in the first round trip, this stays as efficient as today. If there is no such TLS extension, the first response has to include the full polyfill, which is wasteful.
> * If `Sec-CH-UA` includes the actual browser by default, services like `polyfill.io` just need to inspect the `Sec-CH-UA` value to determine what response to send.

Reply:
I don't think the explainer is the place to explore those different options. I prefer to land this current language, discuss the use cases on the various issues and then potentially change the developer advice portion. If that's not something you find reasonable, I can eliminate the developer advice bits for now, and add them once we've come to a consensus on the right approach.

Reply:
Having developer advice is definitely better than not. I think the explainer *is* a good place to explore or describe the consequences of different options, but maybe only in the [Considered Alternatives](https://w3ctag.github.io/explainers#considered-alternatives) section. Feel free to merge this and think about a Considered Alternatives section later.

Reply:
SG! Thanks :)